src/applications/claims-status/components/StemClaimListItem.jsx

```diff
@@ -0,0 +1,45 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Link } from 'react-router';
+import moment from 'moment';
+
+export default function StemClaimListItem({ claim }) {
+  if (!claim.attributes.automatedDenial) {
+    return null;
+  }
+  const formattedReceiptDate = moment(claim.attributes.submittedAt).format(
+    'MMMM D, YYYY',
+  );
+  return (
+    <div className="claim-list-item-container">
+      <h2 className="claim-list-item-header-v2 vads-u-font-size--h3">
+        Your Edith Nourse Rogers STEM Scholarship Application
+        <br />
+        updated on {moment(claim.attributes.deniedAt).format('MMMM D, YYYY')}
+      </h2>
+      <div className="card-status">
+        <div className={`status-circle closed`} />
+        <p>
+          <strong>Status:</strong> Denied
```

Comment:
I have no idea what the endpoint data looks like for these claims, but will status for these always be `Denied`? Is there a way to derive this value from whatever claims data is returned to the frontend?

Reply:
Right now we'll only be displaying denied claims. That might change in the future but I don't want to put in anything that wouldn't be testable by QA (since they couldn't test that it would display correctly for a non-denied claim ATM)